### PR TITLE
Updated dependencies

### PR DESCRIPTION
**Pull request by sheyaln**
Updated http to ^1.2.0
lints to ^3.0.0
mocktail to ^1.0.3

All tests pass. Expedient review would be greatly appreciated.

### CODE REVIEW

**Comment by illia-romanenko** [comment]
Hi @Milksheyke - thanks for your PR! It looks like we'll need to bump Dart as well for this PR (see failed check). Do we have a good reason for doing that since those look as dev dependencies?

**Comment by sheyaln** [comment]
@illia-romanenko Thank you for your response. 
Using newer versions of Shelf or the stable release of Dart Frog requires newer versions of the http package. That was the main goal behind my PR, mocktail and lints were incidental. Please excuse my inexperience and do correct me if I'm wrong, but I believe the version bump for mocktail and lints are needed for the http version bump.

**Comment by illia-romanenko** [comment]
Ok - that's reasonable. Can you fix CI so it's not failing?

**Comment by sheyaln** [comment]
@illia-romanenko I changed only what needed to be changed per Dart's warnings.
Let me know if you'd rather I use `// ignore` on the private type/public API warnings instead of converting them to public.

File: pubspec.yaml [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
-version: 4.11.0
+version: 4.11.1
 description: A simple Stripe API wrapper, that is meant to be used on the server.
 homepage: https://github.com/enyo/stripe-dart
 documentation: https://github.com/enyo/stripe-dart/blob/main/README.md
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: ">3.0.0"
 dependencies:
+  http: ^1.2.0
```
Comment (illia-romanenko) [inline pubspec.yaml]: Do we need http here? We use dio in the package and it's probably not used at all.

**Review by illia-romanenko — CHANGES_REQUESTED** [review]
It looks good - thanks - just 1 minor question and should be ready to go.

File: pubspec.yaml (hunk 1 above)
Comment (sheyaln) [reply pubspec.yaml]: Fair point

**Pull request closed without merging**